backend/people/migrations/0004_fill_staff_data.py

```diff
@@ -0,0 +1,242 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2020-10-22 19:47
+from __future__ import unicode_literals
+from django.db import migrations
+
+def forwards(apps, schema_editor):
+    Staff = apps.get_model('people', 'Staff')
+    Staff.objects.create(
+        name='Alireza Ilami'
```

Comment:
Some of this year staff, already exist in database. So it's better not to create another object for them.